docs/1.34/faq/prisma-1-maintenace-fq12.mdx

```diff
@@ -6,4 +6,42 @@ export const meta = {
 
 ## FAQ
 
-Prisma is currenly in [maintenance mode](https://github.com/prisma/prisma1/issues/4898). There will be no new features developed for Prisma 1.
+Prisma 1 Open Source is currently in [maintenance mode](https://github.com/prisma/prisma1/issues/4898). Prisma will not develop any new features or support future implementations.
+
+### What is the difference between Prisma 1 Open Source and Prisma 1 Cloud?
+
+**Prisma 1 Open Source** is the [client](https://v1.prisma.io/docs/1.34/prisma-client/), [CLI](https://v1.prisma.io/docs/1.34/prisma-cli-and-configuration/), and [server](https://v1.prisma.io/docs/1.34/prisma-server/).
+
+**Prisma 1 Cloud** was a [service](https://app.prisma.io) that complemented Prisma 1 Open Source by hosting the [server](https://v1.prisma.io/docs/1.34/prisma-server/) and a data browser.
+
+### How can I check whether I'm using Prisma 1 Cloud?
+
+1. Open `prisma.yml` in your project.
+2. Find the `endpoint` property.
+3. If the `endpoint` property includes either `https://us1.prisma.sh` or `https://eu1.prisma.sh`, then you are using Prisma 1 Cloud.
+
+### How can I log into Prisma 1 Cloud?
+
+Visit [app.prisma.io](https://app.prisma.io) and log in with an email and password or through Github. After logging in, navigate to the "Services" tab at the top of the screen to access any currently active services, the "Servers" tab for any custom servers you may have added, and "Settings" in the top center of the screen to access billing and integration information.
```

Comment:
A few comments:

- I suggest to use bold font for any UI elements we expect folks to click on
- Since this paragraph is basically a list of instructions, should we maybe make it a numbered list as well? (Similar to the previous FAQ)
- The answer seems to go way beyond the question itself ("How to log into Prisma 1 Cloud") 

```suggestion
Visit [app.prisma.io](https://app.prisma.io) and log in with an email and password or through GitHub. After logging in, navigate to the **Services** tab at the top of the screen to access any currently active services, the **Servers** tab for any custom servers you may have added, and **Settings** in the top center of the screen to access billing and integration information.
```